examples/sveltekit/package.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "sveltekit_example",
+	"version": "0.0.1",
+	"private": true,
+	"scripts": {
+		"dev": "vite dev",
```

Comment:
Jeg spurte ChatGPT om hvorfor sveltekit legger til "dev" men ikke "start" og den påsto at "start" er for å starte produksjons-server og at det visst nok kan gjøres via `svelte-kit start"
```suggestion
		"start": "svelte-kit start",
		"dev": "vite dev",
```
Jeg vet ikke om dette funker, men det kan jo være en idé å legge til et "start" skript fordi "start" er litt spesiell i den forstand at man kan forkorte `npm run start` til `npm start`

Reply:
`svelte-kit start` er ikke en gyldig kommando
![image](https://user-images.githubusercontent.com/43886029/223687242-c82a5890-6f8c-4a23-b7ab-9ef748ce4c24.png)
 
`svelte-kit --help` gir denne outputten:
![image](https://user-images.githubusercontent.com/43886029/223687521-503adc9c-24b1-4973-bb98-9437c3ccbcd0.png)
altså innholder svelte-kit cli'en bare en sync command.

Er det kanskje denne kommandoen du er ute etter?
![image](https://user-images.githubusercontent.com/43886029/223688072-97897a5c-9a71-4c52-9ce5-c6e2e7fb1313.png)

Reply:
@orsolli har du lyst til å lage en eksempel komponent?